apps/condo/domains/organization/components/EmployeeRoleForm/CreateEmployeeRoleForm.tsx

```
@@ -1,16 +1,15 @@
 import get from 'lodash/get'
-import uniqBy from 'lodash/uniqBy'
 import React, { useCallback, useMemo } from 'react'
 
 
 import { useOrganization } from '@open-condo/next/organization'
 
 import { Loader } from '@condo/domains/common/components/Loader'
-import { B2BAppContext, B2BAppPermission, B2BAppRole } from '@condo/domains/miniapp/utils/clientSchema'
 import { UseEmployeeRolesPermissionsGroups } from '@condo/domains/organization/hooks/useEmployeeRolesPermissionsGroups'
 import { OrganizationEmployeeRole as EmployeeRole } from '@condo/domains/organization/utils/clientSchema'
+import { useFilteredB2BAppsData } from '@condo/domains/organization/hooks/useFilteredB2BAppsData'
```

ESLint

[error] `@condo/domains/organization/hooks/useFilteredB2BAppsData` import should occur before import of `@condo/domains/organization/utils/clientSchema`

(import/order)